### PR TITLE
Fixes #17163: After server install, the "no user" login page link to a 404 documentation page

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/17163

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/Login.scala [hunk 1]
```diff
@@ -9,18 +9,26 @@ import scala.xml.NodeSeq
...
   def mainDispatch = Map(
     "display" -> { authForm:NodeSeq =>
       if(userListProvider.authConfig.users.isEmpty) {
         <div class="col-xs-12" style="min-height:250px">
-          <div class="row" style="margin-top:50px">
-            <h3 class="text-danger text-center col-xs-10 col-xs-offset-1">No user is defined.</h3>
+          <div class="row" style="margin-top:30px">
+            <h3 class="text-center">
+              Rudder installation complete!
+              <i class="fa fa-check-circle" style="color: #9bc832;" aria-hidden="true"></i>
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/Login.scala:20]: I would put it on the left side of "Rudder installation complete!"

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/Login.scala [hunk 2]
```diff
@@ -9,18 +9,26 @@ import scala.xml.NodeSeq
...
+            </h3>
+            <div class="text-center" style="margin-top:20px">
+                To get started, create a first user with:
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/Login.scala:23]: I would have put a blue `<i class="fa fa-info-circle" aria-hidden="true"></i>` on the left here

**Comment by ElaadF** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/Login.scala [hunk 3]
```diff
@@ -9,18 +9,27 @@ import scala.xml.NodeSeq
 class Login extends DispatchSnippet with DefaultExtendableSnippet[Login] {
 
   val userListProvider = RudderConfig.rudderUserListProvider
-
+//  Rudder server installation is complete! To get started, create a first user with:
```
Comment (fanf) [inline webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/Login.scala:12]: Can you remove comment?

**Comment by ElaadF** [comment]
Commit modified

**Pull request merged**